docs/content/dagster-cloud/getting-started/getting-started-with-hybrid-deployment.mdx

```diff
@@ -60,17 +62,11 @@ Continue to the next step when finished.
 
 ## Step 3: Deploy your code
 
-Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
-
-A code location specifies a single Python package or file that defines your Dagster code. Each Dagster Cloud deployment can include code from one or more code locations. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
+There are two options for deploying your code. For most Hybrid deployments, with the exception of those using a local agent, you will need to create a Docker image containing your code and then add a code location to Dagster Cloud that references the image. Typically, this build and deploy is done through CI/CD. It is also possible to deploy a code location manually, and this manual process would be used with the local agent.
 
-To complete this step, you can use your own Dagster code or a [project scaffold](/getting-started/create-new-project). Our examples use the scaffold. Scaffold a new project by running:
+- For continunous integration with GitHub, begin by copying the GitHub action deploy workflow in [this repository](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart). You will need to set a variety of secrets as outlined in the action, and should uncomment the step that is relevant to your Docker container registry. You will also want to copy the example Dockerfile. Finally, you should copy the `dagster_cloud.yaml` file and update it accordingly.  If you are not using GitHub but are using a different CI/CD tool, you will want to follow the same steps outlined in the GitHub action, but use the `dagster-cloud` CLI to deploy your code location. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
```

Comment:
we're also desperately in need of a configuration reference for dagster_cloud.yaml

Reply:
Since this contains several sequential steps, it might be good to format it as a list rather than. a paragraph.